skins/oasis/css/core/breakpoints-layout.scss

```diff
@@ -5,9 +5,17 @@
 @if $widthType == 0 {
 	// consider using https://github.com/stowball/mqGenie for release
 	@media #{$breakpoint-small-only} {
+		$content-width: $page-width-small - 2 * $content-border-width;
+
 		.WikiaPage {
 			padding: 0 $padding-width-small;
-			width: $page-width-small - 2 * $content-border-width;
+			width: $content-width;
+		}
+
+		.wds-global-footer {
```

Comment:
Shouldn't this be in https://github.com/Wikia/app/blob/dev/extensions/wikia/DesignSystem/styles/design-system.scss?